.github/workflows/builder.yml

```diff
@@ -52,7 +52,7 @@ jobs:
           rsync -Pav "$image_gz" "$image_sha1" deploy@download.ole.org:/data/images/
           if release_is_number; then
             echo "Marking release as latest image"
-            ssh deploy@download.ole.org:/data/images/ -c ":; cd /data/images; ln -sf $name.img.gz latest.img.gz; ln -sf $name.img.gz.sha1 latest.img.gz.sha1"
+            ssh deploy@download.ole.org -c ":; cd /data/images; ln -sf $name.img.gz latest.img.gz; ln -sf $name.img.gz.sha1 latest.img.gz.sha1"
```

Comment:
Looks just like the original: https://github.com/rjpadilla/builder/blob/5ee774080d96140718b6f6669941dd28773029a1/.travis/upload.sh#L11 💯 💯 💯